### PR TITLE
Build integration e2e

### LINKED ISSUE #2153: e2e test for `okteto build`
Implement an e2e test for the new `okteto build` command

### PR DESCRIPTION
**Pull request by teresaromero**
Fixes #2153 

## Proposed changes

- integration test for okteto build from manifest

### CODE REVIEW

**Review by pchico83 — APPROVED** [review]

File: integration/build_test.go [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+}
+
+func writeManifestV2(path, content string, vars *manifestV2TemplateVars) error {
+	oFile, err := os.Create(path)
+	if err != nil {
+		return err
+	}
+	defer oFile.Close()
+
+	template := template.Must(template.New("manifest_v2").Parse(content))
```
Comment (jpf-okteto) [inline integration/build_test.go:129]: Avoid shadowing package names if not needed.
```suggestion
	tmpl := template.Must(template.New("manifest_v2").Parse(content))
```

File: integration/build_test.go [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+			t.Fatal(err)
+		}
+		defer os.RemoveAll(dir)
+
+		manifestPath := filepath.Join(dir, "okteto.yml")
+		manifestContent := `build:
+  react-getting-started:
+    context: .
+`
+		if err := writeManifestV2(manifestPath, manifestContent, nil); err != nil {
```
Comment (jpf-okteto) [inline integration/build_test.go:88]: Why using template to write this file? It doesn't require customizations.

```go
err := ioutil.WriteFile(manifestPath, []byte(manifestContent), 0644)
```

File: integration/build_test.go [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+		// setup namespace for test
+		startNamespace := getCurrentNamespace()
+		defer changeToNamespace(ctx, oktetoPath, startNamespace)
+
+		if err := createNamespace(ctx, oktetoPath, namespace); err != nil {
+			t.Fatal(err)
+		}
+		defer deleteNamespace(ctx, oktetoPath, namespace)
+
+		// clone repo
```
Comment (jLopezbarb) [inline integration/build_test.go:66]: I don't think this comment is needed, it's already known by function name

**Review by jLopezbarb — APPROVED** [review]

File: integration/build_test.go (hunk 1 above)
Comment (teresaromero) [reply integration/build_test.go]: woops thanks! fixed

Comment (jpf-okteto) [reply integration/build_test.go]: fixed at 924ace5

**Review by jpf-okteto — APPROVED** [review]

File: integration/build_test.go (hunk 2 above)
Comment (teresaromero) [reply integration/build_test.go]: you are right, i removed the templating, but kept as a func because i would need it for other test

Comment (jpf-okteto) [reply integration/build_test.go]: fixed at ce54789

**Pull request merged**